src/encord_active/cli/utils/typer.py

```diff
@@ -0,0 +1,159 @@
+        if panel_name == rutils.ARGUMENTS_PANEL_TITLE:
+            # Already printed above
+            continue
+        rutils._print_options_panel(
+            name=panel_name,
+            params=arguments,
+            ctx=ctx,
+            markup_mode=markup_mode,
+            console=console,
+        )
+
+    for panel_name, options in panel_to_options.items():
+        if panel_name == rutils.OPTIONS_PANEL_TITLE:
+            # Already printed above
+            continue
+        rutils._print_options_panel(
+            name=panel_name,
+            params=options,
+            ctx=ctx,
+            markup_mode=markup_mode,
+            console=console,
+        )
+
+    # MOVED DOWN
+    # default_options = panel_to_options.get(rutils.OPTIONS_PANEL_TITLE, [])
+    # rutils._print_options_panel(
+    #     name=rutils.OPTIONS_PANEL_TITLE,
+    #     params=default_options,
+    #     ctx=ctx,
+    #     markup_mode=markup_mode,
+    #     console=console,
+    # )
+    # MOVE END
+
+    if isinstance(obj, click.MultiCommand):
+        panel_to_commands: DefaultDict[str, List[click.Command]] = defaultdict(list)
+        for command_name in obj.list_commands(ctx):
+            command = obj.get_command(ctx, command_name)
+            if command and not command.hidden:
+                panel_name = getattr(command, rutils._RICH_HELP_PANEL_NAME, None) or rutils.COMMANDS_PANEL_TITLE
+                panel_to_commands[panel_name].append(command)
+
+        # Print each command group panel
+        default_commands = panel_to_commands.get(rutils.COMMANDS_PANEL_TITLE, [])
+        rutils._print_commands_panel(
+            name=rutils.COMMANDS_PANEL_TITLE,
+            commands=default_commands,
+            markup_mode=markup_mode,
+            console=console,
+        )
+        for panel_name, commands in panel_to_commands.items():
+            if panel_name == rutils.COMMANDS_PANEL_TITLE:
+                # Already printed above
+                continue
+            rutils._print_commands_panel(
+                name=panel_name,
+                commands=commands,
+                markup_mode=markup_mode,
+                console=console,
+            )
+
+    # MOVED START
+    default_options = panel_to_options.get(rutils.OPTIONS_PANEL_TITLE, [])
+    rutils._print_options_panel(
+        name=rutils.OPTIONS_PANEL_TITLE,
+        params=default_options,
+        ctx=ctx,
+        markup_mode=markup_mode,
+        console=console,
+    )
+    # MOVE END
+
+    # Epilogue if we have it
+    if obj.epilog:
+        # Remove single linebreaks, replace double with single
+        lines = obj.epilog.split("\n\n")
+        epilogue = "\n".join([x.replace("\n", " ").strip() for x in lines])
+        epilogue_text = rutils._make_rich_rext(text=epilogue, markup_mode=markup_mode)
+        console.print(Padding(Align(epilogue_text, pad=False), 1))
+
+
+rutils.rich_format_help = rich_format_help_ea
```

Comment:
nothing like the good old monkey patch when submitting a PR is too complicated 😆